docs/integrations/digitalocean/k8s.mdx

```diff
@@ -0,0 +1,281 @@
+
+:::caution This how-to guide requires:
+
+1. An [ngrok account](https://ngrok.com/signup) at any tier.
+1. A [DigitalOcean account](https://cloud.digitalocean.com/registrations/new).
+1. [doctl](https://docs.digitalocean.com/reference/doctl/) installed locally.
+1. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed locally.
+
+:::
+
+## **Step 1**: Provision a new cluster on DigitalOcean with `doctl` {#provision-a-new-cluster}
+
+In this guide, we'll stick with the CLI as much as possible to showcase how tightly the ngrok Kubernetes Operator can integrate with DigitalOcean and how you might automate these steps in the future.
+
+1. With `doctl` installed remotely, scope out the node sizes/specs, which will directly affect your [monthly pricing](https://www.digitalocean.com/pricing/kubernetes), with the following:
+
+   ```bash
+   doctl kubernetes options sizes
+   ```
+
+1. Provision your cluster with the command below, replacing `NODE_SIZE` and `CLUSTER_NAME` with your chosen size and a relevant name.
+
+   ```bash
+   doctl kubernetes clusters create --size NODE_SIZE CLUSTER_NAME --1-clicks ngrok-ingress-controller
+   ```
+
+   This command not only provisions your cluster, but also installs the [ngrok Kubernetes Operator](https://github.com/ngrok/kubernetes-ingress-controller) from their [1-Click catalog](https://marketplace.digitalocean.com/apps/ngrok-ingress-controller).
+
+   After a few moments, `doctl` will let you know it's finished deploying the cluster and adding its credentials to your `kubcetl` config file.
+
+   ```bash
+   ...
+   Notice: Successfully kicked off addon job.
+   ID                                      Name        Region    Version        Auto Upgrade    Status     Node Pools
+   d62a17ca-32e0-4d8b-9260-2d0d1c582939    ngrokker    nyc1      1.29.1-do.0    false           running    ngrokker-default-pool
+   ```
+
+1. You can now verify that the cluster is running successfully by looking at the pods in the new `ngrok-ingress-controller` namespace... wait, `CreateContainerConfigError`?
+
+   ```bash
+   kubectl get pods -n ngrok-ingress-controller
+
+   NAME                                                              READY   STATUS                       RESTARTS   AGE
+   ngrok-ingress-controller-kubernetes-ingress-controller-mannvs9b   0/1     CreateContainerConfigError   0          6m30s
+   ```
+
+   That's actually to be expected at this point. If you run `kubectl describe pods -n ngrok-ingress-controller YOUR_NGROK_INGRESS_CONTAINER`, you'd see the following error: `kubelet            Error: secret "ngrok-auth" not found`. Let's remedy that.
```

Comment:
Oh! I see. 😆 While I think the question on the previous line is clever, I think for the tone of this doc, I would take it out and start line 72 with something like. "The `CreateContainerConfigError` is actually to be expected...".

Reply:
I got a little too cheeky! Made the change as suggested.